optaplanner-test/src/main/java/org/optaplanner/test/api/score/stream/ConstraintVerifier.java

```diff
@@ -60,6 +60,16 @@
     ConstraintVerifier<ConstraintProvider_, Solution_> withConstraintStreamImplType(
             ConstraintStreamImplType constraintStreamImplType);
 
+    /**
+     * Applies only to {@link ConstraintStreamImplType#DROOLS}.
+     * Do not enable when running in a native image.
+     *
+     * @param droolsAlphaNetworkCompilerEnabled true to enable the alpha network compiler
+     * @return this
+     */
+    ConstraintVerifier<ConstraintProvider_, Solution_> withDroolsAlphaNetworkCompiler(
```

Comment:
if the field is called CompiLATION, this needs to be called ...ation too. And visa versa.